gulpfile.js

```diff
@@ -74,7 +75,7 @@ gulp.task('connect', function() {
 gulp.task('open', function() {
   return gulp.src('./sandbox/index.html')
     .pipe(open({
-      uri: 'http://localhost:8080',
+      uri: 'http://localhost:8081',
```

Comment:
In case the intent is not to conflict with ports used by any of our other projects, 8081 is used by render